### ISSUE TITLE
Bias - Variance Analysis - DIAGNOSTIC

### ISSUE THREAD

**Issue by vasthelium** [vasthelium/TruSic#15]
Create a diagnostic tool to monitor the model for Overfitting using validation loss curves.
